src/handle_query_contract_ui.c

```diff
@@ -8,7 +8,7 @@ static void set_native_token_stake_ui(ethQueryContractUI_t *msg, context_t *cont
 
     char ftm_ticker[MAX_TICKER_LEN] = "FTM";
     if (memcmp(msg->network_ticker, ftm_ticker, 3) == 0) {
-        context->ticker = "FTM ";
+        context->ticker = &ftm_ticker[0];
```

Comment:
Why not keeping "FTM" like on other spots?
Or maybe this?
```suggestion
        context->ticker = ftm_ticker;
```

Reply:
Indeed, I changed it. :+1: